src/internal/packager/helm/repo.go

```diff
@@ -379,7 +380,7 @@ func (h *Helm) buildChartDependencies() error {
 	return nil
 }
 
-func (h *Helm) loadAndValidateChart(location string) (loader.ChartLoader, *chart.Chart, error) {
+func loadAndValidateChart(location string) (loader.ChartLoader, *chart.Chart, error) {
```

Comment:
Note: Does `*chart.Chart` have to be a reference or can we pass back a copy?

Also, I think we can get away without a doccomment here, it's fairly self-documenting

Reply:
Technically it does not, but since Helm expects to receive a *chart.Chart it feels weird to dereference then reference again